packages/neutron-sdk/src/bindings/query.rs

```diff
@@ -108,4 +117,11 @@ pub struct QueryInterchainAccountAddressResponse {
     pub interchain_account_address: String,
 }
 
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+pub struct QueryFailuresResponse {
+    /// **failures** is a list of failures of sudo handler calls
+    pub failures: Vec<Failure>,
```

Comment:
I think if we return a list of entities it would be nice to return a `PageResponse` in some way in future. We have a lot of PageRequest fields which are useless without PageResponse - `key`,`count_total` for example

Reply:
Let's do it in the next release